### DISCUSSION TITLE
Transaction missing

### DISCUSSION THREAD

**Discussion by TLT4444** [bisq-network/bisq#6746] [Q&A]
Hello,
For some reason, my transaction from Bisq went "missing"?
Transaction:
aaad4ad965f4aa3fc6da7e9bf935e9d93ded9b4eae46640b3d3d74a3710ae94f
I waited for a day but it still kept staying that way.
So I saw  the following post here:
https://github.com/bisq-network/bisq/discussions/6344
I did resync my SPV-File and now the transaction is completely missing from Bisq with the btc too.
Should I do a backup now or what would be the next best step?

**Comment by pazza83** [comment]
Now you have done an SPV resync your wallet should be correctly synced.

It is always good to do regular backups.

But you should be able to use Bisq as normal now.

Your previous transaction failed, so no bitcoin from it would have left your wallet.

**Comment by TLT4444** [comment]
Thank you for the confirmation, I created a new transaction flawlessly.
